core/src/main/java/io/grpc/internal/DelayedClientTransport.java

```diff
@@ -346,7 +344,8 @@ private PendingStream(PickSubchannelArgs args) {
       this.args = args;
     }
 
-    private void createRealStream(ClientTransport transport) {
+    /** Runnable may be null. */
```

Comment:
Why not use `@Nullable` or `@CheckForNull` annotation rather than comment?

Reply:
All of those are equivalent, and I wouldn't spend any time debating which to use for a private method. If we had a nullness checker such that the annotations weren't just a comment with a different syntax, it would be different.